modules/hcp-upgrading-requirements.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * hosted-control-planes/hcp-updating.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hosted-control-planes-upgrading-requirements_{context}"]
+= Requirements to update hosted control planes
+
+The multicluster engine for Kubernetes operator (MCE) can manage one or more {product-title} clusters. After you create a hosted cluster on {product-title}, you must import your hosted cluster in the MCE as a managed cluster. Then, you can use the {product-title} cluster as a management cluster.
+
+Consider the following requirements before you start the update process:
+
+* You must use the bare metal platform for an {product-title} cluster.
+
+* An {product-title} cluster must use the installer-provisioned infrastructure.
+
+* You must use bare metal or {VirtProductName} as the cloud platform for the hosted cluster. You can find the platform type of your hosted cluster in the `spec.Platform.type` specification of the `HostedCluster` resource.
+
+* To upgrade an {product-title} cluster, MCE, hosted cluster, and node pools, complete the following steps:
+
+. Upgrade the MCE version from 2.4 to 2.5.
+. Upgrade an {product-title} cluster from {product-title} 4.14 to {product-title} 4.15.
+. Upgrade the hosted cluster and node pools from {product-title} 4.14 to {product-title} 4.15.
```

Comment:
hosted cluster change to `HostedCluster`
node pools change to `NodePool`

Reply:
I think here it's better if we do not call out the resources name. Because essentially it talks about components rather than resources. In the later sections, updating procedures for these components are described, so IMO we can skip resource names here.